streaming/base/dataset.py

```diff
@@ -139,7 +137,7 @@ def __init__(self,
                  download_retry: int = 2,
                  download_timeout: float = 60,
                  validate_hash: Optional[str] = None,
-                 shuffle_seed: Optional[int] = None,
+                 shuffle_seed: Optional[int] = 9176,
```

Comment:
Type is no longer optional I think

Reply:
Done in https://github.com/mosaicml/streaming/pull/103/commits/69425859864b6df1340636f4eb65245f7ac84538.